doc/dadbod.txt

```diff
@@ -81,6 +81,33 @@ Handling these parameters is up to the individual adapters.  By default,
 unrecognized parameters are ignored.  Pass Boolean parameters as the literal
 strings "true" and "false" (e.g., `?ssl=true`).
 
+                                                *dadbod-bigquery*
+bigquery ~
+>
+    bigquery://[?<globalflag>=<value>][&<globalflag>=<value>][...]
+    bigquery://?format=sparse select count(*) from publicdata.samples.shakespeare
+    bigquery://?dataset_id=publicdata:samples&format=json select count(*) from shakespeare
+    bigquery:[project]// select count(*) from publicdata.samples.shakespeare
+    bigquery:[project:dataset]// select count(*) from publicdata.samples.shakespeare
```

Comment:
Shouldn't need the trailing `//` on these.

Reply:
Removed 👍